test/scripts/models/page.js

```diff
@@ -18,11 +18,11 @@ describe('Page', () => {
     }).then(data => {
       data.title.should.eql('');
       data.date.valueOf().should.gte(now);
-      data.updated.valueOf().should.gte(now);
       data.comments.should.be.true;
       data.layout.should.eql('page');
       data._content.should.eql('');
       data.raw.should.eql('');
+      should.not.exist(data.updated);
```

Comment:
Practically `data.updated` won't be null, thanks to https://github.com/hexojs/hexo/blob/2146a03a2cfced62ecfb623e5da99f21269711f6/lib/plugins/processor/post.js#L90.